iwyu_include_picker.h

```diff
@@ -67,9 +67,17 @@ struct IncludeMapEntry;
 
 enum IncludeVisibility { kUnusedVisibility, kPublic, kPrivate };
 
+struct MappedInclude {
+  explicit MappedInclude(const string& quoted_include);
+
+  string quoted_include;
```

Comment:
Generally I am of the opinion that it's unwise to make struct or class fields `const`, because it suppresses the assignment operator, which means you cannot e.g. sort a `vector` of those objects.

Reply:
Ah, I see. Yeah, makes sense. You could probably work around it with some `= delete` and `= default`, but I think that would add unnecessary noise.